### PR TITLE
docs(helper): update helper docstrings

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
Update the docstrings of public functions and classes in helper

### CODE REVIEW

File: jina/helper.py [hunk 1]
```diff
@@ -92,12 +98,28 @@ def call_obj_fn(obj, fn: str):
 
 
 def touch_dir(base_dir: str) -> None:
+    """
+    Create a directory from given path if it doesn't exist.
+
+    :param base_dir: Path of target path.
+    :returns: None
```
Comment (maximilianwerk) [inline jina/helper.py:105]: Just out of curiosity: If there is no return value, should this still be present?

File: jina/helper.py [hunk 2]
```diff
@@ -92,12 +98,28 @@ def call_obj_fn(obj, fn: str):
...
+    """
     if not os.path.exists(base_dir):
         os.makedirs(base_dir)
 
 
 def batch_iterator(data: Iterable[Any], batch_size: int, axis: int = 0,
                    yield_slice: bool = False, yield_dict: bool = False) -> Iterator[Any]:
+    """
+    Get an iterator of batches of data.
```
Comment (maximilianwerk) [inline jina/helper.py:114]: Can we add one example usage?

File: jina/helper.py [hunk 3]
```diff
@@ -141,6 +163,12 @@ def batch_iterator(data: Iterable[Any], batch_size: int, axis: int = 0,
 
 
 def parse_arg(v: str) -> Optional[Union[bool, int, str, list, float]]:
+    """
+    Parse the arguments from string to list.
```
Comment (maximilianwerk) [inline jina/helper.py:167]:
```suggestion
    Parse the arguments from string to `Union[bool, int, str, list, float]`.
```

File: jina/helper.py [hunk 4]
```diff
@@ -141,6 +163,12 @@ def batch_iterator(data: Iterable[Any], batch_size: int, axis: int = 0,
...
+
+    :param v: The string of arguments
+    :return: The parsed arguments list.
```
Comment (maximilianwerk) [inline jina/helper.py:170]:
```suggestion
    :returns: The parsed arguments.
```

File: jina/helper.py [hunk 5]
```diff
@@ -197,6 +225,11 @@ def random_name() -> str:
 
 
 def random_port() -> Optional[int]:
+    """
+    Get a random available port number from '49153' to '65535'.
+
+    :return: A random port.
```
Comment (maximilianwerk) [inline jina/helper.py:231]:
```suggestion
    :returns: A random port.
```

File: jina/helper.py [hunk 6]
```diff
@@ -225,7 +258,8 @@ def _get_port(port=0):
 
 
 def random_identity(use_uuid1: bool = False) -> str:
-    """Generate random UUID
+    """
+    Generate random UUID.
 
     :param use_uuid1: use UUID1 instead of UUID4. This is the default Document ID generator.
```
Comment (maximilianwerk) [inline jina/helper.py:264]: `:returns:` missing

File: jina/helper.py [hunk 7]
```diff
@@ -240,10 +274,22 @@ def random_identity(use_uuid1: bool = False) -> str:
 
 
 def random_uuid(use_uuid1: bool = False) -> uuid.UUID:
+    """
+    Get a random uuid.
+
+    :param use_uuid1: Use uuid1 if True, else use uuid4.
+    :return: A random uuid.
```
Comment (maximilianwerk) [inline jina/helper.py:281]:
```suggestion
    :returns: A random uuid.
```

File: jina/helper.py [hunk 8]
```diff
@@ -240,10 +274,22 @@ def random_identity(use_uuid1: bool = False) -> str:
...
+    """
     return uuid.uuid1() if use_uuid1 else uuid.uuid4()
 
 
 def expand_env_var(v: str) -> Optional[Union[bool, int, str, list, float]]:
+    """
+    Expand the environment variables.
+
+    :param v: String of environment variables.
+    :return: Parsed environment variables.
```
Comment (maximilianwerk) [inline jina/helper.py:291]:
```suggestion
    :returns: Parsed environment variables.
```

File: jina/helper.py [hunk 9]
```diff
@@ -369,6 +415,12 @@ def build_url_regex_pattern():
 
 
 def is_url(text):
+    """
+    Check if the text is url.
+
+    :param text: The target text.
+    :return: True if text is url else False.
```
Comment (maximilianwerk) [inline jina/helper.py:422]:
```suggestion
    :returns: True if text is URL else False.
```

**Review by maximilianwerk — CHANGES_REQUESTED** [review]
Some `:returns:` are still missing. Also some functions are not yet documented.

**Comment by Yongxuanzhang** [comment]
> Some `:returns:` are still missing. Also some functions are not yet documented.

Sure I will add them. I don't why my local style check didn't pick them out.

File: jina/helper.py (hunk 9 above)
Comment (Yongxuanzhang) [reply jina/helper.py]: Done

File: jina/helper.py (hunk 8 above)
Comment (Yongxuanzhang) [reply jina/helper.py:322]: Done

File: jina/helper.py (hunk 5 above)
Comment (Yongxuanzhang) [reply jina/helper.py:260]: Done

File: jina/helper.py (hunk 2 above)
Comment (Yongxuanzhang) [reply jina/helper.py:121]: Done

File: jina/helper.py (hunk 1 above)
Comment (Yongxuanzhang) [reply jina/helper.py]: Removed

File: jina/helper.py [hunk 10]
```diff
@@ -75,6 +75,12 @@ def wrapper(*args, **kwargs):
 
 
 def get_readable_size(num_bytes: Union[int, float]) -> str:
+    """
+    Transform the bytes into readable value with different units.
+
+    :param num_bytes: Number of bytes.
+    :returns: Transformed bytes.
```
[the remark this answers is not in the document]
Comment (Yongxuanzhang) [reply jina/helper.py]: Done

File: jina/helper.py [hunk 11]
```diff
@@ -75,6 +75,12 @@ def wrapper(*args, **kwargs):
 
 
 def get_readable_size(num_bytes: Union[int, float]) -> str:
+    """
+    Transform the bytes into readable value with different units.
```
[the remark this answers is not in the document]
Comment (Yongxuanzhang) [reply jina/helper.py]: Added

**Pull request merged**